app/view-plants/styles.ts

```diff
@@ -13,3 +14,51 @@ export const TopRowContainer = styled.div`
   margin-top: 8px;
   margin-bottom: 8px;
 `;
+
+export const HeaderButton = styled.button<{ $isCurrentMode: boolean }>`
+  background: none;
+  border: none;
+  color: ${COLORS.shrub};
+  font-family: inherit;
+  cursor: pointer;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
```

Comment:
note, this should be a `styled(P1)`, which would handle the font styling for you. then, you can use HeaderButton with `as="button"`, i.e.  
`<HeaderButton as="button"> </HeaderButton>`

Pull the latest commits on main to see the P1 style in `styles/text.ts`!